compiler/src/dotty/tools/dotc/reporting/diagnostic/ErrorMessageID.java

```diff
@@ -113,6 +113,7 @@
     IllegalStartOfStatementID,
     TraitIsExpectedID,
     TraitRedefinedFinalMethodFromAnyRefID,
+    InvalidUnapplyReturnTypeID,
```

Comment:
Or `UnapplyInvalidReturnTypeID` ? I am now struggling with this for the other one, and decided that `UnapplyInvalidNumberOfArgumentsID` sounds better than `InvalidUnapplyNumberOfArgumentsID`. WDYT?

Reply:
OK for me, I updated the names.